.github/workflows/ci_tests.yaml

```diff
@@ -79,6 +79,14 @@ jobs:
             pandas-version: ''
             xarray-version: ''
             optional-packages: ' contextily geopandas ipython pyarrow rioxarray sphinx-gallery'
+          # The job below is for testing geopands v0.x on Ubuntu. Python 3.11 is used
+          # here, otherwise Python 3.10/3.12 jobs will be override by this one.
```

Comment:
```suggestion
          # The job below is for testing geopandas v0.x on Ubuntu. Python 3.11 is used
          # here, otherwise Python 3.10/3.12 jobs will be overwridden by this one.
```